src/frontend/src/components/tables/settings/UserTable.tsx

```diff
@@ -35,13 +38,88 @@ export interface UserDetailI {
   is_superuser: boolean;
 }
 
+export function UserDrawer({
+  id,
+  refreshTable
+}: {
+  id: string;
+  refreshTable: () => void;
+}) {
+  const {
+    instance: userDetail,
+    refreshInstance,
+    instanceQuery: { isFetching, error }
+  } = useInstance<UserDetailI>({
+    endpoint: ApiPaths.user_list,
+    pk: id,
+    throwError: true
+  });
+
+  if (isFetching) {
+    return <LoadingOverlay visible={true} />;
+  }
+
+  if (error) {
+    return (
+      <Text>
+        {(error as any)?.response?.status === 404 ? (
+          <Trans>User with id {id} not found</Trans>
+        ) : (
+          <Trans>An error occurred while fetching user details</Trans>
+        )}
+      </Text>
+    );
+  }
+
+  return (
+    <Stack>
+      <EditApiForm
```

Comment:
I find this way of selecting the roles far less intuitive than the pills that signified no to all access from left to right. The texts are also not fitting as we use the roles in other ways.
![grafik](https://github.com/inventree/InvenTree/assets/66015116/f3302ddb-8109-49a5-a6ea-476e88a6be3f)

Reply:
I can change the texts, that's no problem. But look at the lines of code required for the user drawer now. That's by far less than the old implementation and easier to understand. And there is even more functionality than in the old implementation. Editing a user other than the permissions works now. I actually find it better to have some description instead of having no explanation.

Reply:
@SchrodingersGat what is your opinion on this one?

Reply:
I don't think that "active" / "staff" / "superuser" can be classified under the same banner as "rights".

I prefer the left hand screenshot here - but I'd like to see a comparison for group management too

Reply:
No, only the name of the group can be changed for now. Adjusting the rule set is not possible.

Reply:
There are now 2 ways to change user data and group data and over-complicated info text with references to sites the user probably doe not need to know about - I do not see how this is an improvement in usability.

Reply:
What do you mean with two ways? And what empty detail page?

Reply:
There are 2 ways to edit user details: the row action and the drawer; same applies to groups;
![grafik](https://github.com/inventree/InvenTree/assets/66015116/c47ee2a0-0d87-4219-840f-0015c4f37423)
1 actionable field that is already exposed through row actions => empty